experimenter/templates/experimenter_base.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    {% block title %}
+    {% endblock %}
```

Comment:
Instead of this, lets do:

```django
<title>{% block title %}{% endblock %}</title>
```

and then we don't have to repeat `<title>` every time we inherit from this template.